### PR TITLE
test: updated metadata as per network name

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* test: updated metadata as per the network name

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**Changes**

* test: updated metadata as per the network name

### CODE REVIEW

**Review by sirius651 — APPROVED** [review]
LGTM

File: src/pages/XvmTransfer.vue [hunk 1]
```diff
@@ -8,11 +8,20 @@ import { defineComponent } from 'vue';
 import { useMeta } from 'quasar';
 import XvmTransfer from 'src/components/assets/transfer/XvmTransfer.vue';
 import { usePageReady } from 'src/hooks';
+import { meta } from 'src/config/metadata';
 
 export default defineComponent({
   components: { XvmTransfer },
   setup() {
-    useMeta({ title: 'XVM Transfer' });
+    useMeta({
```
Comment (bobo-k2) [inline src/pages/XvmTransfer.vue:16]: Maybe to define a function which accepts title and returns metadata object. I see very similar code on lot of places.

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

**Comment by Kahonnohak** [comment]
Sorry but please change "Astar" in all descriptions on Shiden to "Shiden"
スクリーンショット 2023-02-08 13 17 12

File: src/pages/XvmTransfer.vue (hunk 1 above)
Comment (impelcrypto) [reply src/pages/XvmTransfer.vue]: @bobo-k2  Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/689/commits/cc70f25e2c0f95d6c4778730d193097a551c256a) commit.

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Comment by impelcrypto** [comment]
@Kahonnohak 
> Sorry but please change "Astar" in all descriptions on Shiden to "Shiden"

Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/689/commits/590c50ba317ca33f8c304fe864f3c3bcf2ff92e2) commit

**Review by Kahonnohak — APPROVED** [review]
LGTM thanks!

**Pull request merged**